src/Classes/Controllers/AdminController.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Portal\Controllers;
+
+use Slim\Views\PhpRenderer;
+
+class AdminController
+{
+    private $renderer;
+
+    function __construct(PhpRenderer $renderer)
+    {
+        $this->renderer = $renderer;
+    }
+
+    function __invoke($request, $response, $args)
```

Comment:
docblock

Reply:
disagree, slim calls this function so dont actually need the docblock as never used by programmer